src/services_models/services_constructor.jl

```diff
@@ -463,3 +462,48 @@ function construct_service!(
     add_constraint_dual!(container, sys, model)
     return
 end
+
+function construct_service!(
+    container::OptimizationContainer,
+    sys::PSY.System,
+    ::ArgumentConstructStage,
+    model::ServiceModel{T, ConstantMaxInterfaceFlow},
+    devices_template::Dict{Symbol, DeviceModel},
+    incompatible_device_types::Set{<:DataType},
+) where {T <: PSY.TransmissionInterface}
+    # contributing_devices_map = get_contributing_devices_map(model)
```

Comment:
Should these commented-out lines go away? Or maybe add a comment about why they are here?